backend/dal/Services/Admin/Concrete/BaseService`.cs

```diff
@@ -46,7 +34,9 @@ public virtual ET Find(params object[] keyValues)
         {
             this.User.ThrowIfNotAuthorized(Permissions.SystemAdmin, Permissions.AgencyAdmin);
 
-            return this.Context.Set<ET>().Find(keyValues);
+            var result = this.Context.Set<ET>().Find(keyValues);
+            this.Context.Entry(result).State = Microsoft.EntityFrameworkCore.EntityState.Detached; // Force detach so that outside the DAL the DB cannot be manipulated.
```

Comment:
I think this only detaches the top level entity.

Reply:
Good point.  I should update the comment.

Reply:
I also experimented with manually parsing all entities in context and removing them one-by-one. This threw a number of weird errors for me, which I believe is because removing entities from context in this manner is not a supported workflow:
https://github.com/dotnet/efcore/issues/18406#issuecomment-543984411

Officially the way to remove all tracking is to dispose of the active dbcontext. This won't always work for us because we get our dbcontext injected into the controller action, so I'm not sure what the best way forward is.

Reply:
The best way would probably be not passing Entities outside of the DAL.  But I don't want to do that atm.